### PR TITLE
XW-1086 | Mercury should still work even with cookies disabled

### PR DESCRIPTION
**Pull request by hakubo**
## Links

* https://wikia-inc.atlassian.net/browse/XW-1086
* https://github.com/Modernizr/Modernizr/blob/master/feature-detects/storage/localstorage.js

## Description

You can't actually call window.localStorage when cookies are disabled. It'll throw. So you have to do a check in try/catch. 

## Reviewers
@Wikia/x-wing

### CODE REVIEW

File: front/main/app/routes/application.js [hunk 1]
```diff
@@ -112,7 +120,11 @@ export default Ember.Route.extend(
 				 */
 				if (target.className.indexOf('external') > -1) {
 					if (target.href.indexOf(`useskin=${defaultSkin}`) > -1) {
-						document.cookie = `useskin=${defaultSkin}${cookieDomain}; path=/`;
+						$.cookie('useskin', defaultSkin, {
+							domain: cookieDomain,
+							expires: date,
```
Comment (rogatty) [inline front/main/app/routes/application.js]: `date` is undefined here.

**Comment by rogatty** [comment]
LGTM

**Comment by hakubo** [comment]
I saw you walking a minute ago, are you doing CR through phone ? :)

File: front/main/app/utils/local-storage-connector.js [hunk 2]
```diff
@@ -55,9 +55,22 @@ export const localStorageAdapter = {
 	}
 };
 
+// Check if we have local storage available
+let localStorageAvailable = false;
```
Comment (bognix) [inline front/main/app/utils/local-storage-connector.js]: minor: i would scope it inside method

Comment (hakubo) [inline front/main/app/utils/local-storage-connector.js]: taken offline

variables inside module are scoped only to this module anyway

**Comment by hakubo** [comment]
Tested manually.

**Pull request merged**